actions/installer.yml

```diff
@@ -0,0 +1,27 @@
+name: slsa-verifier-installer
+description: 'Installs SLSA verifier and adds it to your PATH'
+branding:
+  icon: 'package'
+  color: 'blue'
+inputs:
+  verifier-release:
+    description: 'slsa-verifier release version to install'
+    required: false
+    default: 'v1.2.0'
+  install-dir:
+    description: 'Absolute path to installation directory'
+    required: false
+    default: '/usr/bin/'
+  use-sudo:
```

Comment:
If we only support `/usr/bin`, we can remove this option. If there's a directory that does not require root (`opt/` ?), we could use it instead.

Reply:
follow-up: A folder under the user's home directory may be a better option with the need for sudo'ing, like `~/.slsa/bin/`

Reply:
Done